packages/default-theme/pages/account.vue

```diff
@@ -1,79 +1,84 @@
-        <SfContentPage title="Logout"></SfContentPage>
-      </SfContentPages>
+    <SfContentPages
+      title="My account"
+      :active="activePage"
+      @click:change="updateActivePage"
+    >
+      <SfContentCategory title="Personal Details">
+        <SfContentPage title="My profile">
+          <MyProfile />
+        </SfContentPage>
+        <SfContentPage title="My addresses">
+          <SfTabs :open-tab="1">
+            <MyAddresses />
+          </SfTabs>
+        </SfContentPage>
+        <SfContentPage title="My newsletter">
+          <SfTabs :open-tab="1">
+            <SfTab title="My newsletter"> </SfTab>
+          </SfTabs>
+        </SfContentPage>
+      </SfContentCategory>
+      <SfContentCategory title="Order details">
+        <SfContentPage :title="`Order history (${user && user.orderCount})`">
+          <OrderHistory />
+        </SfContentPage>
+      </SfContentCategory>
+      <SfContentPage title="Logout"></SfContentPage>
+    </SfContentPages>
   </div>
 </template>
 <script>
-import { SfContentPages, SfTabs, SfList } from "@storefront-ui/vue"
-import { useUser } from "@shopware-pwa/composables"
-import MyProfile  from "../components/account/MyProfile"
-import MyAddresses from "../components/account/MyAddresses"
-import OrderHistory from "../components/account/OrderHistory"
+import { SfContentPages, SfTabs, SfList } from '@storefront-ui/vue'
+import { useUser } from '@shopware-pwa/composables'
+import MyProfile from '../components/account/MyProfile'
+import MyAddresses from '../components/account/MyAddresses'
+import OrderHistory from '../components/account/OrderHistory'
 export default {
   name: 'Account',
-  components: { OrderHistory, SfContentPages, SfTabs, MyProfile, SfList, MyAddresses },
-  middleware: "auth",
+  components: {
+    OrderHistory,
+    SfContentPages,
+    SfTabs,
+    MyProfile,
+    SfList,
+    MyAddresses
+  },
+  middleware: 'auth',
   setup() {
     const { logout, user, loadOrders, orders } = useUser()
     return { logout, user, loadOrders, orders }
   },
   data() {
     return {
-      activePage: "My profile",
+      activePage: 'My profile',
       allAddresses: []
     }
   },
   computed: {
     activeBillingAddress() {
-      return this.user && this.user && this.user.activeBillingAddress || {}
+      return (this.user && this.user && this.user.activeBillingAddress) || {}
     },
     activeShippingAddress() {
-      return this.user && this.user && this.user.activeShippingAddress || {}
-    },
+      return (this.user && this.user && this.user.activeShippingAddress) || {}
+    }
   },
   // async mounted() {
   //   this.allAddresses = await this.getAddresses()
   // },
   methods: {
     async updateActivePage(title) {
-        if (title === "Logout") {
-          await this.logout();
-          this.$router.push("/login")
-        }
-        this.activePage = title;
+      if (title === 'Logout') {
+        await this.logout()
+        this.$router.push('/login')
```

Comment:
please make sure to use  theme helpers pages for this:
https://github.com/DivanteLtd/shopware-pwa/pull/357/files#diff-7a540de2ec1a4bb3248db50b16ff00e3R3